packages/synapse-react-client/src/components/SynapseTable/SynapseTableUtils.ts

```diff
@@ -114,3 +115,22 @@ export function isSortableColumn(column: ColumnType) {
       return false
   }
 }
+
+export function getDefaultPrimaryKey(
```

Comment:
Extracted behavior from useTableRowSelection. In a future commit, this will be used, and the logic that this duplicates will be removed from useTableRowSelection.